engine/gamesys/src/java/com/defold/window/WindowJNI.java

```diff
@@ -26,21 +26,11 @@ public WindowJNI(Activity activity) {
     }
 
     public void enableScreenDimming() {
-        this.activity.runOnUiThread(new Runnable() {
-            @Override
-            public void run() {
-                activity.getWindow().clearFlags(android.view.WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON);
-            }
-        });
+        this.activity.runOnUiThread(() -> activity.getWindow().clearFlags(android.view.WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON));
```

Comment:
somehow an anonymous class failed the build, but a lambda didn't. There is also a warning `Type java.lang.Runnable was not found`, probably related